lib/common.sh

```diff
@@ -32,15 +32,10 @@ is_sbt_native_packager() {
 	fi
 }
 
-_has_build_properties_file() {
-	local ctx_dir="${1}"
-	test -e "${ctx_dir}/project/build.properties"
-}
-
 get_supported_sbt_version() {
 	local ctx_dir="${1}"
 	local sbt_version_pattern="${2:-${SBT_0_VERSION_PATTERN}}"
-	if _has_build_properties_file "${ctx_dir}"; then
+	if test -e "${ctx_dir}/project/build.properties"; then
```

Comment:
```suggestion
	if [[ -f "${ctx_dir}/project/build.properties" ]]; then
```

(for consistency with other spots where you test for files)